.github/pull_request_template.md

```diff
@@ -1,43 +1,8 @@
-<!--
-Not all PRs will require all tests to be carried out. Delete where appropriate.
--->
-
-<!--
-MODIFY THIS AFTER your new app repo is in https://github.com/giantswarm/github
-@team-halo-engineers will be automatically requested for review once
-this PR has been submitted. (But not for drafts)
--->
-
 This PR:
 
 - adds/changes/removes etc
 
-### Testing
-
-Description on how {APP-NAME} can be tested.
-
-- [ ] fresh install works
-  - [ ] AWS
-  - [ ] Azure
-  - [ ] KVM
-- [ ] upgrade from previous version works
-  - [ ] AWS
-  - [ ] Azure
-  - [ ] KVM
-
-#### Other testing
-
-Description of features to additionally test for {APP-NAME} installations.
-
-- [ ] check reconciliation of existing resources after upgrading
-- [ ] X still works after upgrade
-- [ ] Y is installed correctly
-
-<!--
-Changelog must always be updated.
--->
-
 ### Checklist
 
 - [ ] Update changelog in CHANGELOG.md.
-- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
+- [ ] Make sure to run `make generate` to generate Helm manifests.
```

Comment:
Not now, but we should add tests that ensure that `make generate` was executed, something like this in CAPI https://github.com/kubernetes-sigs/cluster-api/blob/v1.0.3/Makefile#L699-L704.